.travis.yml

```diff
@@ -29,8 +29,12 @@ script:
     # Run unit tests and doc tests in debug mode
     - cargo test -p lumol-core
     - cargo test -p lumol-input
+    # Check if tutorials compile
+    - cd
```

Comment:
Looks like a previous attempt leftover, but this might break the build, as the code is not in $HOME.